src/renderers.jl

```diff
@@ -36,6 +36,16 @@ function render(doc::MultiDocRef, dir, thispagepath, prettyurls)
     """
 end
 
+function render(c::Link, doc, thispage, prettyurls)
+    # class nav-link nav-item makes the formatting correct
+    # _target="blank" opens the link in a new tab
+    # TODO: add "external link" icon after, either chain or arrow exiting box.
+    # TODO: allow internal links
+    return @htl """
+    <a href=$(c.link) class="nav-link nav-item" _target="blank">$(c.text)</a>
+    """
```

Comment:
```suggestion
    # class nav-link nav-item makes the formatting correct
    # target="_blank" opens the link in a new tab
    # TODO: add "external link" icon after, either chain or arrow exiting box.
    # TODO: allow internal links
    return @htl """
    <a href=$(c.link) class="nav-link nav-item" target="_blank">$(c.text)</a>
    """
```
I messed this up in the previous comment :sweat_smile: